src/sdk/pynni/nni/compression/torch/apply_compression.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+import torch
+from .compressor import Pruner
+
+logger = logging.getLogger('torch apply compression')
+
+def apply_compression_results(model, masks_file):
+    """
+    Apply the masks from ```masks_file``` to the model
+
+    Parameters
+    ----------
+    model : torch.nn.module
+        The model to be compressed
+    masks_file : str
+        The path of the mask file
+    """
+    apply_comp = ApplyCompression(model, masks_file)
+    apply_comp.compress()
+
+class ApplyCompression(Pruner):
```

Comment:
why subclass Pruner?

Reply:
This class is for validating the correctness of ModelSpeedup, i.e., compare inference result from ModelSpeedup and that from ApplyCompression. ApplyCompression simply applies the masks, from implementation, it can be seen as a simple pruner.

Reply:
I think it can simply be a method to go through all masks and apply to the weight accordingly. Because there is no need to hook forward method like pruner does and all cal_mask does is a dict lookup. To sum up, it is not a pruner and subclassing pruner makes it look weird. A simple function would do exactly the same and make much more sense?

Reply:
good suggestion, let me fix it in a folow-up pr.